enigma-core/app/Cargo.toml

```diff
@@ -25,7 +25,7 @@ log = "0.4.6"
 log-derive = "0.2"
 simplelog = "0.5.3"
 structopt = "0.2"
-
+rustc-demangle = "<0.1.14"
```

Comment:
Can you please rebase this *without* the stuff on `rustc-demangle` and `rust-web3` that were already fixed https://github.com/enigmampc/enigma-core/pull/120